{
  "id": "EUVD-2021-1465",
  "enisaUuid": "dfa2599c-f5f6-396e-b7af-4b53967edc21",
  "description": "This affects the package pimcore/pimcore before 10.0.7. This issue exists due to the absence of check on the storeId parameter in the method collectionsActionGet and groupsActionGet method within the ClassificationstoreController class.",
  "datePublished": "Jul 9, 2021, 12:40:15 PM",
  "dateUpdated": "Sep 16, 2024, 7:04:05 PM",
  "baseScore": 8.3,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:L/E:F/RL:O/RC:C",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2021-23405",
    "https://github.com/pimcore/pimcore/pull/9572",
    "https://snyk.io/vuln/SNYK-PHP-PIMCOREPIMCORE-1316297"
  ],
  "aliases": [
    "CVE-2021-23405",
    "GHSA-g8jx-66p8-vcm2"
  ],
  "assigner": "snyk",
  "epss": 0.03,
  "enisaIdProduct": [],
  "enisaIdVendor": [
    {
      "id": "750ea6ab-ec12-3a05-b692-0cb63d2fe459",
      "vendor": {
        "name": "n/a"
      }
    }
  ]
}